src/arrayMethodSort.js

```diff
@@ -5,7 +5,26 @@
  */
 function applyCustomSort() {
   [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+    for (let i = 0; i < this.length; i++) {
+      for (let j = i + 1; j < this.length; j++) {
+        if (this[j].toString() < this[i].toString()) {
+          const temp = this[i];
+
+          this[i] = this[j];
+          this[j] = temp;
+        }
+
+        if (compareFunction !== undefined
+          && compareFunction(this[j], this[i]) < 0) {
+          const temp = this[i];
+
+          this[i] = this[j];
+          this[j] = temp;
+        }
```

Comment:
so you can swap elements twice?
You should use `this[j].toString() < this[i].toString()` as a default param for `compareFunction` to have smth like
```suggestion
        if (compareFunction(this[j], this[i]) < 0) {
          const temp = this[i];

          this[i] = this[j];
          this[j] = temp;
        }
```